lumigator/python/mzai/backend/schemas/deployments.py

```diff
@@ -28,3 +28,7 @@ class DeploymentEvent(BaseModel):
     deployment_type: DeploymentType
     status: DeploymentStatus
     detail: str | None = None
+
+
+class DeploymentEventList(RootModel):
```

Comment:
we should only need to inherit from `BaseModel `. https://docs.pydantic.dev/latest/concepts/models/#model-methods-and-properties

Reply:
Ok, thanks. This was an attempt at loading lists of X, until I noticed https://github.com/mozilla-ai/lumigator/blob/b1971690e0b9abf45280c69831f141be7164d468/lumigator/python/mzai/backend/schemas/extras.py#L21